# RFID状态

Header header

# 基地增益点
bool base
# 高地增益点
bool elevated
# 能量机关激活点
bool powerRune
# 飞坡增益点
bool launchRamp
# 前哨站增益点
bool outpost
# 补血点增益点
bool restoration
# 工程机器人补血卡
bool engineer